conf/flash_modes.xml

```diff
@@ -47,6 +47,7 @@
       <board name="li[s]?a_mx_.*"/>
       <board name="px4fmu_.*"/>
       <board name="elle*"/>
+      <board name="lia_.*"/>
```

Comment:
that's what I thought, but frankly I didn't spend much time figuring it out - adding lia explicitly made the trick